extensions/cql/examples-array.txt

```diff
@@ -0,0 +1,285 @@
+
+0. beamMode is ScanSAR Narrow AND swathDirection is ascending AND
+   polarization is "HH+VV+HV+VH" and intersects geom(Washington DC)
+
+CQL: beamMode='ScanSAR Narrow' AND
+     swathDirection='ascending' AND 
+     polarization='HH+VV+HV+VH' AND
+     intersects(geometry,POLYGON((-77.117938 38.936860,-77.040604 39.995648,-76.910536 38.892912,-77.039359 38.791753,-77.047906 38.841462,-77.034183 38.840655,-77.033142 38.857490)))
+
+CQL JSON: [
+             "and",
+              ["eq", ["property", "beamMode"], "ScanSAR Narrow"],
+              ["eq", ["property", "swathDirection"], "ascending"],
+              ["eq", ["property", "polarization"], "HH+VV+HV+VH"],
+              ["intersects",
+                  ["property", "footprint"],
+                  {
+                      "type": "Polygon",
+                      "coordinates": [[
+                          [-77.117938,38.936860],
+                          [-77.040604,39.995648],
+                          [-76.910536,38.892912],
+                          [-77.039359,38.791753],
+                          [-77.047906,38.841462],
+                          [-77.034183,38.840655],
+                          [-77.033142,38.857490]
+                      ]]
+                  }
+             ]
+          ]
+
+1. Floors greater than 5
+
+CQL: floors>5
+
+CQL JSON: [
+              "eq",
+              ["property", "floor"],
+              5
+          ]
+
+2. Taxes less than or equal to 500
+
+CQL: taxes <= 500
+
+CQL JSON: [
+              "lte",
+              ["property", "taxes"],
+              500
+          ]
+
+3. Owner name contains 'Jones'
+
+CQL: owner LIKE '% Jones %'
```

Comment:
```suggestion
CQL: owner_name LIKE '% Jones %'
```

Reply:
Since `owner` and `owner_name` were both used in the existing example, I proposed a change to make them consistent in #285.